tests/cypress/integration/webhooks.js

```diff
@@ -0,0 +1,101 @@
+// Copyright (C) 2022 CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+context('Webhooks pipeline.', () => {
+    const organizationParams = {
+        shortName: 'WebhooksOrg',
+        fullName: 'Organization full name. Only for test.',
+        description: 'This organization was created to test the functionality.',
+        email: 'testorganization@local.local',
+        phoneNumber: '+70000000000',
+        location: 'Country, State, Address, 000000',
+    };
+    const orgWebhookParams = {
+        targetURL: 'https://localhost:3001/organization',
+        description: 'Sample description',
+        secret: 'Super secret',
+        enableSSL: true,
+        isActive: true,
+        events: [
+            'project', 'job', 'task',
+        ],
+    };
+    const projectWebhookParams = {
+        targetURL: 'https://localhost:3001/project',
+        description: 'Sample description',
+        secret: 'Super secret',
+        enableSSL: true,
+        isActive: true,
+    };
+    const newProjectWebhookParams = {
+        targetURL: 'https://localhost:3001/edited',
+        description: 'Edited description',
+        secret: 'Super secret',
+        enableSSL: true,
+        isActive: false,
+        events: [
+            'job',
+        ],
+    };
+
+    const project = {
+        name: 'Project for webhooks',
+        label: 'car',
+        attrName: 'color',
+        attrVaue: 'red',
+        multiAttrParams: false,
+    };
+
+    before(() => {
+        cy.visit('auth/login');
+        cy.login();
+        cy.createOrganization(organizationParams);
```

Comment:
We need to remove the created organization after tests are finished in ``after``